cypress-tests/cypress/e2e/v2/sessionBasics.cy.ts

```diff
@@ -0,0 +1,195 @@
+import { TIMEOUTS } from "../../../config";
+import {
+  getRandomString,
+  getUserData,
+  validateLoginV2,
+} from "../../support/commands/general";
+import { ProjectIdentifierV2 } from "../../support/types/project.types";
+import { User } from "../../support/types/user.types";
+import {
+  createProjectIfMissingAPIV2,
+  deleteProjectFromAPIV2,
+  getProjectByNamespaceAPIV2,
+} from "../../support/utils/projectsV2.utils";
+
+const sessionId = ["sessionBasics", getRandomString()];
+
+describe("Start a session that consumes project resources", () => {
+  // Define required resources
+  let projectNameRandomPart: string;
+  let projectName;
+  let projectIdentifier: ProjectIdentifierV2;
+
+  function resetRequiredResources() {
+    projectNameRandomPart = getRandomString();
+    projectName = `session-basics-${projectNameRandomPart}`;
+    projectIdentifier = {
+      slug: projectName,
+      id: null,
+      namespace: null,
+    };
+  }
+
+  beforeEach(() => {
+    // Restore the session (login)
+    cy.session(
+      sessionId,
+      () => {
+        cy.robustLogin();
+      },
+      validateLoginV2,
+    );
+
+    // Create a project
+    resetRequiredResources();
+    getUserData().then((user: User) => {
+      projectIdentifier.namespace = user.username;
+      createProjectIfMissingAPIV2({
+        name: projectName,
+        namespace: user.username,
+        slug: projectName,
+      }).then((project) => (projectIdentifier.id = project.id));
+    });
+  });
+
+  // Cleanup the project after the test
+  after(() => {
+    getProjectByNamespaceAPIV2(projectIdentifier).then((response) => {
+      if (response.status === 200) {
+        projectIdentifier.id = response.body.id;
+        projectIdentifier.namespace = response.body.namespace;
+        deleteProjectFromAPIV2(projectIdentifier);
+      }
+    });
+  });
+
+  it("Start a session with VSCode and check it has the necessary resources", () => {
+    const dataConnectorName = `giab-${getRandomString()}`;
+    const repositoryName = "renku-ui";
+    const session = {
+      arguments:
+        '["/entrypoint.sh jupyter server --ServerApp.ip=0.0.0.0 --ServerApp.port=8888 --ServerApp.base_url=$RENKU_BASE_URL_PATH --ServerApp.token=\\"\\" --ServerApp.password=\\"\\" --ServerApp.allow_remote_access=true --ContentsManager.allow_hidden=true --ServerApp.allow_origin=* --ServerApp.root_dir=\\"/home/jovyan/work\\""]',
+      command: '["sh","-c"]',
+      image: "registry.renkulab.io/rok.roskar/vscode-example:0c10599",
+      name: "vscode-launcher",
+      url: "/vscode",
```

Comment:
I would use `renku/renkulab-vscodium-python-runimage:ubuntu-59fcea4` which is an official image from the Renku team and not something hosted at `registry.renkulab.io/rok.roskar`.

That image also has simpler arguments: `command=["bash"]`, `arguments=["/entrypoint.sh"]`, `workdir=/home/ubuntu/work`, `mountdir=/home/ubuntu/work`

Reply:
Oh, good suggestion! I only had to change a few other details and it works well :+1: 
https://github.com/SwissDataScienceCenter/renku/pull/3912/commits/1097b60a1f96f059e8693ed9636821e202ea4a14